base/doc/ltnews39.tex

```diff
@@ -187,6 +187,18 @@ \subsection{Warn if counter names are problematic}
 %
 \githubissue{823}
 
+\subsection{Optimise creation of simple document commands}
+
```

Comment:
We should go with the US spelling (which is Optimize)

Reply:
Corrected: thought I'd got all of these